src/js/flags/debug-mode.ts

```diff
@@ -1,6 +1,6 @@
 import {FlagSection} from './flag';
 
-export const debugModeFlags: FlagSection = {
+export const DEBUG_MODE_FLAGS: FlagSection = {
```

Comment:
I'm not paticularly married to the ALL_CAPS for const, though it's consistent with the rest of the codebase, and would be a lot of work to change them all consistently.  In particular, all the enums under graph/ would be nice to bring into line with TypeScript standards (UpperCamel).  For now it's probably best to keep them all consistent tho, and if I ever get around to automating a change then it might get better.

Reply:
I've seen it as the typical way for doing constants across a lot of JS/Java projects (it's what we use at work as well). I'm open to whatever as long as it's consistent.